projects/graph/src/structures.py

```diff
@@ -0,0 +1,90 @@
+class Queue:
+    def __init__(self):
+        self.size = 0
+        self.storage = LinkedList()
+
+    def enqueue(self, item):
+        self.storage.add_to_tail(item)
+
+    def dequeue(self):
+        if self.storage.head is None:
+            return None
+        else:
+            old_head = self.storage.head.value
+            self.storage.remove_head()
+            return old_head
```

Comment:
It was smart putting the queue structure in another file and importing it so you can keep your file sizes down.